app/concepts/movies/use_case/create.rb

```diff
@@ -0,0 +1,7 @@
+module UseCase
+  module Movie 
+    class Create < UseCase::Base:Create
```

Comment:
Is this working?

Reply:
Im going push commit in 30 minutes with fixes. There were bugs with inheritance, I misstyped some things.